guppy/src/graph/build.rs

```diff
@@ -548,7 +546,7 @@ impl PackageSourceImpl {
         let path_diff = if path_diff.is_absolute() {
             path_diff
         } else {
-            convert_forward_slashes(path_diff)
+            convert_relative_forward_slashes(path_diff)
```

Comment:
Since convert_forward_slashes() will convert relative path and clone absolute path,
these 6 lines of code can be done as 1 line:

Self::Path(convert_relative_forward_slashes(path_diff).into_boxed_path())